docs/toast/index.html

```diff
@@ -123,8 +123,8 @@ <h3>Toast</h3>
           </div>
           <div class="navigation-footer-links">
             <div class="prev-page">
-              <a href="../rating/"
-                ><i class="fas fa-chevron-left"></i>&nbsp;Rating</a
+              <a href="../toast/"
+                ><i class="fas fa-chevron-left"></i>&nbsp;Toast</a
```

Comment:
liked the clear and precise commit messages. helps the reviewer to understand what exactly to look at.